erehwon/profiles/forms.py

```diff
@@ -1,7 +1,6 @@
 from django import forms
 from profiles.models import Project
-
-from profiles.models import ErehwonUser, Project
+from profiles.models import ErehwonUser
 
 from registration.forms import RegistrationForm
```

Comment:
You probably want to add both of the project forms defined here as well, so only the fields you want the users to edit can be editable rather than all. For example which user the project belongs to should not be editable by the user